test/unit/ConstantProductPair.t.sol

```diff
@@ -726,6 +735,7 @@ contract ConstantProductPairTest is BaseTest, IReservoirCallee {
         // act - swap once at half volume, again with platform fee.
         vm.prank(address(_factory));
         _constantProductPair.setCustomPlatformFee(type(uint256).max);
+        _constantProductPair.burn(address(this));
```

Comment:
fyi this we need this statement to set the `kLast = 0`, so that the next burn actually mints the platform fee. Without this, the next `burn` would set `kLast = 0`, resulting in no new shares minted, which would make the `assertLt` trivially true

Reply:
Sounds good